ecs-cli/modules/cli/compose/project/project_test.go

```diff
@@ -17,331 +17,50 @@ import (
-	assert.Equal(t, shmSize, wordpress.ShmSize, "Expected shmSize to match")
-	assert.Equal(t, tmpfs, wordpress.Tmpfs, "Expected tmpfs to match")
-
-	// verify mysql ServiceConfig
-	mysql, ok := configs.Get("mysql")
-	if mysql == nil || !ok || mysql.Image != mysqlImage {
-		t.Fatalf("Expected [%s] as a service with image [%s] but got configs [%v]", "redis", mysqlImage, configs)
-	}
-}
-
-func TestParseComposeForVersion1WithEnvFile(t *testing.T) {
-	envKey := "rails_env"
-	envValue := "development"
-	envContents := []byte(envKey + "=" + envValue)
+    image: mysql`
 
-	envFile, err := ioutil.TempFile("", "example")
-	if err != nil {
-		t.Fatal("Error creating tmp file:", err)
-	}
-	defer os.Remove(envFile.Name()) // clean up
-	if _, err := envFile.Write(envContents); err != nil {
-		t.Fatal("Error writing to tmp file:", err)
-	}
+	tmpfile, err := ioutil.TempFile("", "test")
+	assert.NoError(t, err, "Unexpected error in creating test file")
 
-	webImage := "webapp"
+	defer os.Remove(tmpfile.Name())
 
-	composeFileString := `web:
-  image: webapp
-  env_file:
-  - ` + envFile.Name()
+	_, err = tmpfile.Write([]byte(composeFileString))
+	assert.NoError(t, err, "Unexpected error in writing to test file")
 
-	// setup project and parse
-	tmpfile, err := ioutil.TempFile("", "test")
-	if err != nil {
-		t.Fatal("Unexpected error in creating test file", err)
-	}
-	defer os.Remove(tmpfile.Name())
+	err = tmpfile.Close()
+	assert.NoError(t, err, "Unexpected error closing file")
 
-	if _, err := tmpfile.Write([]byte(composeFileString)); err != nil {
-		t.Fatal("Unexpected error writing to test file: ", err)
-	}
-	if err := tmpfile.Close(); err != nil {
-		t.Fatal("Unexpected error closing test file: ", err)
-	}
+	// Set up project
 	project := setupTestProject(t)
 	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile.Name())
 
-	if err := project.parseCompose(); err != nil {
-		t.Fatalf("Unexpected error parsing the compose string [%s]: %v", composeFileString, err)
-	}
-
-	configs := project.ServiceConfigs()
-
-	// verify wordpress ServiceConfig
-	web, ok := configs.Get("web")
-	if web == nil || !ok || web.Image != webImage {
-		t.Fatalf("Expected [%s] as a service with image [%s] but got configs [%v]", "redis", webImage, configs)
-	}
+	// verify container configs are populated
+	err = project.parseCompose()
+	assert.NoError(t, err, "Unexpected error parsing file")
+	assert.NotEmpty(t, project.ContainerConfigs(), "Expected container configs to not be empty")
 
-	// skips the second one if envKey2
-	if web.Environment == nil || len(web.Environment) != 1 {
-		t.Fatalf("Expected non empty Environment, but was [%v]", web.Environment)
-	}
-	if string(envContents) != web.Environment[0] {
-		t.Errorf("Expected env [%s]=[%s] But was [%v]", envKey, envValue, web.Environment)
-	}
+	// verify project name is set
+	assert.Equal(t, testProjectName, project.ecsContext.ProjectName, "Expected ProjectName to be overridden.")
```

Comment:
what do we expect the project name to be at this point? the name of the temp directory?

Reply:
This was just a conversion of the assertion that existed [previously](https://github.com/aws/amazon-ecs-cli/pull/507/files#diff-26963a7116c253fb1f9db039a342f726L127).